### ISSUE TITLE
experience of using RUCIO  as base-user

### ISSUE THREAD

**Issue by grahal-esc** [rucio/rucio#4021]
Motivation:  I have experienced some issues in using rucio especially because of some lack of information of what it is doing while executing a command. As promised, I write part of my observations as base-user.
add-rule: trying to replicate a dataset of some 100GB (106 files) and using "list-rules" or "list dataset replicas" or "list files replicas" to follow the progress:
- Replication works for a bunch of files, then the progress is stopped for tens of hours, then suddenly after few days, the dataset is replicated. However the State does not change on list-rules and does not show any "stuck" state.
-  list-files-replicas shows the files of my datasets on the different RSEs but list-dataset-replicas shows always none/ You can check with:  rucio list-dataset-replicas NESSIE:iotests2  and rucio list-file-replicas NESSIE:iotests2
- As comparison, the upload works always fine: rucio -v upload --rse DESY-DCACHE  --protocol root --register-after-upload --scope NESSIE NESSIE:iotests2 *
The most difficult thing I have found is to debug: putting or not debug or verbose does not change anything. there is no information provided. Same is true wether the things go right or wrong.  And I did not find in any documentation or tutorial the way (debug or logs) to know what is going on.

Hope it helps. Thanks, Ghita
----------

Modification
------------

**Comment by bari12** [comment]
Thanks Ghita,
We will look into this and will make some improvements. Either on documentation or on some functionality directly.

For the `list-dataset-replicas` - this is currently an issue on the ESCAPE rucio instance which we weren't able to debug yet - Normally this should of course show you correct information.

Cheers,
Martin

**Comment by grahal-esc** [comment]
Hi Martin,

Just to add up some more info to the issue above. Today, I have added 2 rules to copy files to DESY and EULAK from a dataset that is at CC-IN2P3. First, the add-rule worked perfectly and very fast. I have checked that the files have been properly copied (106 files to each). So list-files-replicas works correctly. And list-dataset-replicas seems now to give me the right number of files for each RSE (not immediately but after some time 1/2h), which was not the case before. So it seems to me that the 2 problems I have had are not here any more. Remains only the info that is provided that is quite "scarce" sometimes. Thank you, Ghita
